src/routes/edit-user-vocabulary/EditUserVocabularyScreen.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { ReactElement, useState } from 'react'
```

Comment:
What was changed here? Not reviewed. I really think you should revert renaming.

Reply:
renamed